float64 omega1
float64 omega2
float64 v3
float64 theta1
float64 theta2
---
geometry_msgs/Vector3  result
